CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
   - Major/Minor/Patch: This is an example changelog item, usually can be the commit message.
   - Append new items to make git merging easier.
   - Patch: Ensure `Calendar` day headings are center aligned
+  - Major: Create new `Number` component and integrate it with `CustomFieldInputNumber`
```

Comment:
Theoretically, if there aren't any breaking changes then it would just be a minor change. But I haven't read through the PR to see what is breaking yet.